packages/js/src/support/store/index.js

```diff
@@ -16,22 +21,26 @@ const createStore = ( { initialState } ) => {
 		actions: {
 			...linkParamsActions,
 			...preferencesActions,
+			setCurrentPromotions,
 		},
 		selectors: {
 			...linkParamsSelectors,
 			...preferencesSelectors,
+			isPromotionActive,
 		},
 		initialState: merge(
 			{},
 			{
 				[ LINK_PARAMS_NAME ]: getInitialLinkParamsState(),
 				[ PREFERENCES_NAME ]: getInitialPreferencesState(),
+				currentPromotions: { promotions: get( window, "wpseoScriptData.currentPromotions", [] ) },
```

Comment:
Please get the current promotions from the script data outside of the store creation.
That way you can keep creating this store without it relying on window.

See `packages/js/src/support/initialize.js` for the initial state from the window/script data.

Reply:
Good point, thanks! 

(I also did the same for the dashboard store for some other things as well. As a result I added a quick smoke test of https://github.com/Yoast/wordpress-seo/pull/21644 for this PR's impact check)